assets/css/main.scss

```diff
@@ -81,7 +81,8 @@ $footer-padding: 3rem 1.5rem 0;
 // in addition to the icon:
 $why-carousel-card-height: 10 * $size-5 * $body-line-height;
 $vertical-section-padding: nth($section-padding, 1);
-$why-carousel-offset: (($why-carousel-card-height) / 2 + $vertical-section-padding);
+// $why-carousel-offset: (($why-carousel-card-height) / 2 + $vertical-section-padding);
```

Comment:
Let's remove that line

Reply:
Whoops, sorry - time pressure. Removed it now.